snow/networking/handler/handler.go

```diff
@@ -360,7 +360,7 @@ func (h *handler) dispatchSync(ctx context.Context) {
 			h.StopWithError(ctx, fmt.Errorf(
 				"%w while processing sync message: %s",
 				err,
-				msg,
+				msg.Op().String(),
```

Comment:
Because these are serious errors - I do think we want to have the error include the full message (not just the message type).

We should think about embedding the `fmt.Stringer` interface to the `message.InboundMessage` interface.

Reply:
The biggest problem currently is that Inbound message's message field can be returned as any.  Specifically, it returns  messages defined by protobuf in proto/p2p/p2p.pb.go, none of which implement stringer.  I'll take a look at protobuf and see if there's a way to regenerate the messages such that they all can implement stringer.

Reply:
All of the proto messages implement stringer. ex: https://github.com/ava-labs/avalanchego/blob/master/proto/pb/p2p/p2p.pb.go#L516